src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -2240,17 +2240,17 @@ PeerImp::onMessage(
 void
 PeerImp::onMessage(std::shared_ptr<protocol::TMValidation> const& m)
 {
-    auto const closeTime = app_.timeKeeper().closeTime();
-
-    if (m->validation().size() < 50)
-    {
-        JLOG(p_journal_.warn()) << "Validation: Too small";
-        fee_ = Resource::feeInvalidRequest;
-        return;
-    }
-
     try
     {
+        auto const closeTime = app_.timeKeeper().closeTime();
+
+        if (m->validation().size() < 50)
+        {
+            JLOG(p_journal_.warn()) << "Validation: Too small";
+            fee_ = Resource::feeInvalidRequest;
+            return;
+        }
```

Comment:
Is there a reason this was moved inside the try block? I don't believe `closeTime` or the other functions can throw.

I don't object, but in this same file we move some code outside of a `try` block. I'm not sure the reason this was was moved in while the other one was moved out.

Reply:
No, it can be moved out.